cogs/commands/summarise.py

```diff
@@ -85,19 +114,30 @@ async def dispatch_api(self, messages) -> Optional[str]:
             reply = clean(reply, "Apollo: ", "apollo: ", name)
         return reply
 
-    async def create_message(self, message_chain, prompt):
+    async def create_message(self, message_chain, prompt, ctx):
         # get initial prompt
         initial = prompt + "\n"
 
         # for each message, append it to the prompt as follows --- author : message \n
+        message_length = 0
         async for msg in message_chain:
             if CONFIG.AI_INCLUDE_NAMES and msg.author != self.bot.user:
-                initial += msg.author.name + ":" + msg.content + "\n"
-
+                message_length += len(msg.content.split())
```

Comment:
Did you want to add the thing to only go back as far as the last tldr (perhaps with an option to override)
What exactly is the point of message_length?